lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "2.11.0",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "0.0.0",
```

Comment:
Will you be using fixed versioning mode? I don't think that will work with having 6.x inquirer in with 7.x/latest

You can use independent versioning mode though: https://github.com/lerna/lerna#independent-mode---independent

Reply:
Ah I thought I initialized it that way, but looks like I messed this up. Updated.